### PR TITLE
CSE release 1.64 chrony src-artifact

### PR DESCRIPTION
**Pull request by mvasl**

### CODE REVIEW

File: modules/470-chrony/images/chrony/werf.inc.yaml [hunk 1]
```diff
@@ -1,58 +1,72 @@
...
+    install:
+    - '**/*.go'
+    - 'go.mod'
+    - 'go.sum'
+shell:
+  beforeInstall:
+  - git clone --depth 1 --branch {{ $version }} {{ $.SOURCE_REPO }}/chrony/chrony.git /src/chrony
+---
 artifact: {{ .ModuleName }}/build-chrony-static-artifact
 from: {{ .Images.BASE_ALPINE_DEV }}
```
Comment (RomanenkoDenys) [inline modules/470-chrony/images/chrony/werf.inc.yaml:22]: Please do not use DEV image, uese BASE_ALPINE with proxy

File: modules/470-chrony/images/chrony/werf.inc.yaml [hunk 2]
```diff
@@ -1,58 +1,72 @@
...
+  - cd /src
+  - CFLAGS="-static" LDFLAGS="-static" ./configure --prefix=/opt/chrony-static
+  - make -j1
+  - make -j1 install
+  - chown -R 64535:64535 /opt/chrony-static
+  - chmod +x /opt/chrony-static/bin/chronyc
+  - chmod +x /opt/chrony-static/sbin/chronyd
 ---
 artifact: {{ .ModuleName }}/build-entrypoint-artifact
 from: {{ .Images.BASE_GOLANG_20_ALPINE }}
```
Comment (RomanenkoDenys) [inline modules/470-chrony/images/chrony/werf.inc.yaml:40]: Please use BASE_GOLANG_23_ALPINE

File: modules/470-chrony/images/chrony/werf.inc.yaml [hunk 3]
```diff
@@ -1,58 +1,72 @@
 {{- $version := "4.4" }}
```
Comment (RomanenkoDenys) [inline modules/470-chrony/images/chrony/werf.inc.yaml:1]: may be upgrade to 4.6 ?

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: modules/470-chrony/images/chrony/werf.inc.yaml (hunk 1 above)
Comment (mvasl) [reply modules/470-chrony/images/chrony/werf.inc.yaml]: Which proxy are you talking about? chrony build has all of its code bundled in git, go build is already using GOPROXY

File: modules/470-chrony/images/chrony/werf.inc.yaml (hunk 2 above)
Comment (mvasl) [reply modules/470-chrony/images/chrony/werf.inc.yaml]: done

File: modules/470-chrony/images/chrony/werf.inc.yaml (hunk 3 above)
Comment (mvasl) [reply modules/470-chrony/images/chrony/werf.inc.yaml]: done

File: modules/470-chrony/images/chrony/werf.inc.yaml (hunk 1 above)
Comment (RomanenkoDenys) [reply modules/470-chrony/images/chrony/werf.inc.yaml]: alpine packages proxy. If you cannot install any packages proxy isn't needed.

Comment (mvasl) [reply modules/470-chrony/images/chrony/werf.inc.yaml]: No packages are installed, base image changed, so, it is done

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**